.github/workflows/jetbrains-auto-update-template.yml

```diff
@@ -37,19 +37,20 @@ jobs:
         with:
           distribution: zulu
           java-version: "11"
-      - name: Leeway build
+      - name: Setup Google Cloud
         uses: google-github-actions/setup-gcloud@v0
         with:
           project_id: ${{ secrets.projectId }}
+      - name: Leeway build
         env:
           LEEWAY_REMOTE_CACHE_BUCKET: gitpod-core-leeway-cache-branch
-      - run: |
+        run: |
           gcloud auth configure-docker --quiet
           export LEEWAY_WORKSPACE_ROOT=$(pwd)
           data=$(curl -sSL "https://data.services.jetbrains.com/products?code=${{ inputs.productCode }}&fields=distributions%2Clink%2Cname%2Creleases&_=$(date +%s)000")
           link=$(echo "$data" | jq -r '.[0].releases[0].downloads.linux.link')
           cd components/ide/jetbrains/image
-          leeway build -Dversion=latest -DimageRepoBase=eu.gcr.io/gitpod-core-dev/build -DjetbrainsBackendQualifier=latest -D${{ inputs.productId }}DownloadUrl=$link .:${{ inputs.productId }}
+          leeway build -Dversion=latest -DimageRepoBase=eu.gcr.io/gitpod-core-dev/build -D${{ inputs.productId }}DownloadUrl=$link .:${{ inputs.productId }}-latest
```

Comment:
I look at leeway package with suffix `-latest`, it doesn't use `${{ inputs.productId }}DownloadUrl`, it is a fixed url, so If we want to make sure this always latest version, we should **not** use cache